web-app/webpack.config.js

```diff
@@ -1,9 +1,13 @@
 const webpack = require('webpack');
 const path = require('path');
+const glob = require('glob-all')
 const config = require('sapper/config/webpack.js');
 const pkg = require('./package.json');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const PurifyCSSPlugin = require('purifycss-webpack');
+const WebpackPwaManifest = require('webpack-pwa-manifest');
 
-const mode = process.env.NODE_ENV;
+const mode = process.env.NODE_ENV || 'production';
 const dev = mode === 'development';
 
 const alias = { svelte: path.resolve('node_modules', 'svelte') };
```

Comment:
I was planning to add an alias to `src` here so that we don't have these painful `../../../` import paths all over the place, can you drop a TODO here when you make the other changes to the file?

Reply:
Added the alias and changed the import statement for the static assets...I figure we can change the rest as we're touching other files.